### PR TITLE
Fixed pytest.

### PR DESCRIPTION
**Pull request by cmcheung-UCS**
Signed-off-by: Ming Cheung <chi.m.cheung@intel.com>

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2164,7 +2164,14 @@ task makeAllArtifacts {
             "makeDockerInstallEventSimConfig", "makeDockerInstallTarball3rdParty", "makeDockerInstallTarballDai",
             "makeDeveloperDockerTarball"
 }
-rootProject.build.finalizedBy(makeAllArtifacts)
+
```
Comment (paul-amonson) [inline build.gradle:2167]: I thought we decided I was doing this.

**Review by paul-amonson — CHANGES_REQUESTED** [review]
REmove build.gradle changes. It need to function in all automation.

File: build.gradle (hunk 1 above)
Comment (cmcheung-UCS) [reply build.gradle:2167]: Note that I did not ask for reviewers.  :)

Comment (cmcheung-UCS) [reply build.gradle:2167]: I was just testing it.

**Pull request closed without merging**